### ISSUE TITLE
Asteroid Base docking does not report landing pad

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#224]
## Steps to reproduce

Visit an Asteroid Base and request docking.

## Expected
Docking Granted script reads out the landing pad number.

## Observed
Nothing is spoken at all.

## Investigation
I've tried both my script and the default EDDI script, both have the same result.
I have added some test code to the Docking Granted script.  Using this to set up variables:
```
{set station to StationDetails(event.station, system.name)}
{set station_model to token(StationDetails(event.station, system.name).model," ",1)}
```
And this to test output:
```
event station: {event.station}.
system: {system.name}.
station model: {station.model}.
short model: {station_model}.
pad: {event.landingpad}.
```
I get this as output:
```
event station: Big Pappa's Base.
system: Andhrimi.
station model:.
short model:.
pad: 10.
```

It seems the model is not populated for the asteroid bases.  This test code works on a normal star base, with the following output:
```
event station: Yurchikhin Station.
system: Andhrimi.
station model: Coriolis Starport.
short model: Starport.
pad: 39.
```

**Comment by Tkael** [comment]
I suspect old server data. [The server data for Andhrimi does not include `Big Pappa's Base`](http://api.eddp.co/systems/Andhrimi), so the function StationDetails() returns only empty values.

**Comment by Darkcyde13** [comment]
\*Sigh\* That's just typical, the one I go to test at, is one that 'doesn't exist' in the database yet.  I guess this is probably why the forum member couldn't get it to work either then.

Thinking back now, I did think it was odd that when I arrived in the system, EDDI said that there were only three orbital stations, when there are four including the asteroid base.  This really should have raised a red flag to me. :(
